### PR TITLE
DDF-1835 - Fixes site list returned when fanout is set to true.

### PR DESCRIPTION
**Pull request by coyotesqrl**
A small regression was introduced recently that ignored fanout. This resolves that and adds two integration tests for verification.

Please note that the new `stopManagedService` method in the `ServiceManager` has not been completely verified. It should work but I wouldn't say with certainty that it does work. It is there for future use for cleanup of itests and needs to be made more robust and verified.

@tbatie - Please hero this
@ryeats 
@kcwire 
@clockard

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/506" target="_blank">ReviewNinja

**Comment by clockard** [comment]
LGTM :+1:  
FYI regression was my fault. Thanks for fixing @coyotesqrl

File: distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java [hunk 1]
```diff
@@ -118,6 +118,21 @@ public void startManagedService(String servicePid, Map<String, Object> propertie
         startManagedService(sourceConfig, properties);
     }
 
+    /**
+     * Stops a managed service.
+     *
+     * @param servicePid
+     * @throws IOException
+     */
+    public void stopManagedService(String servicePid) throws IOException {
```
Comment (kcwire) [inline distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java]: Not sure what happens if you have multiple sources configured of the same type. Does it just stop the first 1?

Comment (coyotesqrl) [inline distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java]: Not sure at all. Right now, I think this method should be considered experimental and untested.

**Comment by tbatie** [comment]
Hero'd successfully!

**Pull request merged**